src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3Topic.cs

```diff
@@ -24,14 +24,15 @@ internal static class Fdc3Topic
     internal static string GetIntentResult => TopicRoot + "getIntentResult";
     internal static string SendIntentResult => TopicRoot + "sendIntentResult";
     internal static string AddIntentListener => TopicRoot + "addIntentListener";
+    public static string ResolverUI => TopicRoot + "resolverUI";
```

Comment:
Could you explain the reason for this being public when everything else is internal?